lib/chef/provider/chef_group.rb

```diff
@@ -15,13 +15,13 @@ def whyrun_supported?
       if differences.size > 0
         description = [ "update group #{new_resource.name} at #{rest.url}" ] + differences
         converge_by description do
-          rest.put("groups/#{new_resource.name}", normalize_for_put(new_json))
+          rest.put("organizations/#{new_resource.org}/groups/#{new_resource.name}", normalize_for_put(new_json))
```

Comment:
This is the recipe snippet I was playing around with. So your saying I should be changing the chef_server_url before calling chef_group lwrp and append `/organizations/blackberry`?

```
chef_server_url = "https://localhost:443"

run_context.cheffish.current_chef_server = {
  :chef_server_url => chef_server_url,
  :options => {
    :client_name => "pivotal",
    :signing_key_filename => "/etc/opscode/pivotal.pem" }}

private_key "/etc/opscode/poliva.pem" do
  public_key_path "/etc/opscode/poliva.pub"
end

chef_user "poliva" do
  display_name "Phil Oliva"
  admin true
  email "poliva@blackberry.com"
  password "passw0rd"
  source_key_path "/etc/opscode/poliva.pem"
end

chef_organization "blackberry" do
  full_name "BlackBerry"
  members "poliva"
end

chef_group "admins" do
  users "poliva"
  org "blackberry"
end
```